### ISSUE TITLE
[Bug]: Because of low contrast its impossible to see the text selection

### ISSUE THREAD

**Issue by skycommand** [Razee4315/MarkLite#146]
### What happened?

The user writes a paragraph and selects a portion of it. But... the user can't see what he has selected because the color contrast is either too low or non-existent. Please see the screenshots below.

### Steps to reproduce

1. Open a new blank document
2. Type out the following paragraph

   > But I must explain to you how all this mistaken idea of denouncing pleasure and praising pain was born and I will give you a complete account of the system, and expound the actual teachings of the great explorer of the truth, the master-builder of human happiness. No one rejects, dislikes, or avoids pleasure itself, because it is pleasure, but because those who do not know how to pursue pleasure rationally encounter consequences that are extremely painful. Nor again is there anyone who loves or pursues or desires to obtain pain of itself, because it is pain, but occasionally circumstances occur in which toil and pain can procure him some great pleasure. To take a trivial example, which of us ever undertakes laborious physical exercise, except to obtain some advantage from it? But who has any right to find fault with a man who chooses to enjoy a pleasure that has no annoying consequences, or one who avoids a pain that produces no resultant pleasure?

3. Select the word "must"
4. Ask a friend to tell you what word you've selected and what it reads. He should be able to tell you, but depending on the theme, he can't do one of the other.

### Paperling version

1.0.49

### Operating system

Windows

### Screenshots / logs (optional)

In these images, I've selected one word. Can you tell me what it is?

<img width="924" height="242" alt="Image" src="https://github.com/user-attachments/assets/4a49274d-2bfc-44f2-8d66-a0e37ed33c42" />

<img width="924" height="252" alt="Image" src="https://github.com/user-attachments/assets/04896415-00f1-4ea9-904d-55786af3a334" />

In these images, you can see what word I've selected. But can you read it?

<img width="924" height="243" alt="Image" src="https://github.com/user-attachments/assets/d2fd6b49-9c41-4e19-914a-87ba1e46e041" />

<img width="924" height="246" alt="Image" src="https://github.com/user-attachments/assets/14bbe0c4-1c3b-451d-8dbd-9017d41d79b0" />
